index.html

```diff
@@ -33,7 +33,156 @@ <h1 id="headline">Hey there. I'm Paul Sáenz
           <li><a href="https://github.com/elafildecolor"><img src="images/github.svg" alt="Github logo" class="social-media-icons"></a> </li>
           <li><a href="https://www.angellist.com/"><img src="images/angellist.svg" alt="Angellist logo" class="social-media-icons"></a> </li>          
         </ul>
-    </section>    
+        <a href="#myRecentWorksContainer" target="_blank" rel="noopener noreferrer"><img src="images/anchorButton.svg" alt="Anchor icon"></a>
+    </section>
+
+    <article id="myRecentWorksContainer">
+      <header>
+        <h1 class="headline">My Recent Works</h1>
+        <hr id="recentWorkSeparator">
```

Comment:
- [OPTIONAL]  I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> 💡 NOTE: you've used the `<h1>` tag once here `<h1 class="headline">Hey there. I'm Paul Sáenz  ... `, and that's great because this is the main title 💯

> **Only Use H1 Elements Once Per Page for optimal SEO**